.circleci/config.yml

```diff
@@ -113,6 +116,9 @@ workflows:
           context:
             - hmpps-common-vars
           cache_key: "v2_0"
+          executor:
+            name: hmpps/java
+            tag: 11.0
```

Comment:
Is [CircleCI complaining](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-incentives-api/241/workflows/5fd459de-e18a-45d8-ae3c-4b312fb39a58/jobs/709) because it's expecting this to be a string, e.g. `tag: "11.0"` rather than a number? 🤔

Reply:
That was half the issue 👍  the other half was I had the executors defined in the wrong place